lib/lib-storage/src/Upload.ts

```diff
@@ -17,7 +18,7 @@ import { EventEmitter } from "events";
 
 import { byteLength } from "./bytelength";
 import { getChunk } from "./chunker";
-import { BodyDataTypes, Options, Progress, ServiceClients } from "./types";
```

Comment:
Mark `ServiceClients` as deprecated, and replace it from `Options` interface too.
https://github.com/aws/aws-sdk-js-v3/blob/cbab94e901aec4650ab9e0eb19a4515e44d4ba62/lib/lib-storage/src/types.ts#L52